pages/admin/topics/[id]/index.vue

```diff
@@ -54,7 +54,7 @@ const { t } = useI18n()
 
 const save = async () => {
   if (await validate()) {
-    await $api(`/api/1/topics/${props.topic.id}/`, {
+    await $api(`/api/2/topics/${props.topic.id}/`, {
```

Comment:
I think we should PUT on topic with api v2. Done in https://github.com/datagouv/cdata/pull/455/commits/b63a50a27fbd7d555fa5e29bb0d3a5b986eb7223